examples/erc20/AAVE/aDAI.xml

```diff
@@ -30,12 +30,12 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         <ts:ethereum contract="aDAI"/>
     </ts:origins>
 
-    <ts:selection name="enabled" filter="allowance>0">
+    <ts:selection name="enabled" filter="${allowance}>0">
```

Comment:
I think the interpolation `${}` only works on the RHS, not LHS

Reply:
@hboon 

> <ts:attribute name="allowance">
>         <ts:type><ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax></ts:type>
>         <ts:origins>
>             <ethereum:call function="allowance" contract="DAI" as="uint">
>                 <ts:data>
>                     <ts:address ref="ownerAddress"/>
>                     <ts:address>0x5d3a536e4d6dbd6114cc1ead35777bab948e3643</ts:address>
>                 </ts:data>
>             </ethereum:call>
>         </ts:origins>
>     </ts:attribute>

Reply:
I mean the original is correct:

```
<ts:selection name="enabled" filter="allowance>0">
```

Interpolation is on the RHS, eg.

Imagine we have:

```
<ts:attribute name="someValue">
    //...
</ts:attribute>
```

Then we can do this on the RHS:

```
<ts:selection name="enabled" filter="allowance>${someValue}">
```